### DISCUSSION TITLE
[Bug report] Git suddenly stop syncing with Github Pages

### DISCUSSION THREAD

**Discussion by alxfsl** [GetPublii/Publii#1707] [Bug Report]
### Operating system

Windows 11 Pro 23H2

### Publii version

Version: 0.46.1 (build 16918)

### Issue type

Application

### Bug description

Git Repository suddenly stopped syncing, first by giving a timeout error, and on the second trial returning a positive syncing message but Github wasn't receiving any push. 
I tried to reset the server. Test Connection in Git with GitHub Pages doesn't work (it get stuck - I saw there is already a report about it). Resetting didn't work anyway, with again a error of timeout first and then false successful syncing messages. 
I tried to change the server using GitHub Pages (glad you didn't remove it yet), but could not connect. 

My solution: duplicate the site, sync the server through GitHub Pages with a new token, and all works again. 
It is not the first time the Git Repo does that, I had already flag it in the old forum. Just wanted to let you know that this problem keeps happening

### Steps to reproduce

I don't know how to reproduce because it was syncing and then suddenly stopped working

**Comment by bjazmoore** [comment]
I wonder if it is a Publii issue or a GitHub issue.  Was there anything in the logs on the old instance of your site before you duplicated it?
